src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -9,13 +9,624 @@
     }) }}
 </div>
 
-{{ include('@WebProfiler/Profiler/base_js.html.twig') }}
-
 <style{% if csp_style_nonce %} nonce="{{ csp_style_nonce }}"{% endif %}>
     {{ include('@WebProfiler/Profiler/toolbar.css.twig') }}
 </style>
-<script{% if csp_script_nonce %} nonce="{{ csp_script_nonce }}"{% endif %}>/*<![CDATA[*/
-    (function () {
-        Sfjs.loadToolbar('{{ token }}');
-    })();
-/*]]>*/</script>
+
+<script>
+    window.addEventListener('DOMContentLoaded', () => {
+        new SymfonyWebDebugToolbar('{{ token }}');
+    });
+
+    class SymfonyWebDebugToolbar {
+        #token;
+        #toolbar;
+        #profilerStorageKey = 'symfony/profiler/';
+        #requestStack = [];
+        #successStreak = 4;
+        #pendingRequests = 0;
+
+        constructor(token) {
+            this.#token = token;
+            this.#initializeCopyToClipboardButtons();
+            {% if excluded_ajax_paths is defined %}
+            /* TODO: remove the comment when the code works:
+            this.#proxifyAjaxRequests();
+            */
+            {% endif %}
+            this.#loadToolbar(this.#token);
+        }
+
+        #initializeCopyToClipboardButtons() {
+            document.querySelectorAll('[data-clipboard-text]').forEach((copyButton) => {
+                copyButton.classList.remove('hidden');
+                copyButton.addEventListener('click', () => {
+                    navigator.clipboard.writeText(copyButton.getAttribute('data-clipboard-text'));
+
+                    if (copyButton.classList.contains('label')) {
+                        const oldContent = copyButton.textContent;
+
+                        copyButton.textContent = '✅ Copied!';
+                        copyButton.classList.add('status-success');
+
+                        setTimeout(() => {
+                            copyButton.textContent = oldContent;
+                            copyButton.classList.remove('status-success');
+                        }, 7000);
+                    }
+                });
+            });
+        }
+
+        #request(url, onSuccess, onError, payload, options, tries) {
```

Comment:
For the toolbar, I strongly suggest sticking to older JS features. Otherwise, we forbid projects to test their own pages in older browsers as the Symfony toolbar would break the JS code.

Reply:
I disagree about this. In my opinion, there are no reasons to have to use an old browser when developing a Symfony app locally. My guess is that 100% of Symfony developers use browsers that support all these features (which are not that advanced, probably only the prehistoric Edge browser doesn't support them).

Reply:
Well, not as your daily browser. But being able to reproduce an issue in another browser to work on a fix is important, and it would be a pain to have to disable the WDT to be able to work on such a fix.